Cargo.toml

```diff
@@ -42,7 +42,7 @@ url = { version = "2.5.3", default-features = false, features = ["std"] }
 cargo = { level = "warn", priority = -1 }
 nursery = { level = "warn", priority = -1 }
 pedantic = { level = "warn", priority = -1 }
-multiple_crate_versions = "allow"
```

Comment:
Why is this removal needed?

Reply:
It's not needed (for this PR), I just stuck it in because we no longer need this `allow`.